.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
still looking for some documentation about what `synchronize` is. It seems not to be just a merge of `master` into the PR's branch but also adding any commit. Will try removing to see what happens.